docs/integrations/prometheus-grafana.md

```diff
@@ -0,0 +1,18 @@
+# Prometheus and Grafana
+
+To ingest metrics into Prometheus, add the metrics exporter endpoint to your Prometheus configuration:
+
+```yaml
+scrape_configs:
+  - job_name: 'gpu_metrics'
+    static_configs:
+      - targets: ['exporter_external_ip:5000']
+```
+
+Pre-built Grafana dashboards are available in the `grafana/` directory of the repository:
+
```

Comment:
We should be referencing the files in https://github.com/ROCm/device-metrics-exporter/tree/main/grafana and not gpu-operator repo here. Also we are missing the Overview Dashboard as there are 4 example dashboards we provide. We should also include example screenshot of each of these dashboards which I can provide.